subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -48,8 +48,42 @@ struct wifi_roaming_params {
 static struct wifi_roaming_params roaming_params;
 #endif
 
+const bool is_enterprise_security(enum wifi_security_type security)
```

Comment:
similar with function is_eap_valid_security, pls move one of them to avoid duplicate impl